std/haxe/macro/TypeTools.hx

```diff
@@ -128,6 +128,7 @@ class TypeTools {
 					[ for (a in args)
 						{
 							name: a.name,
+							namePos: {file: "", min: -1, max: -1}, // TODO, I guess?
```

Comment:
@Simn should we also use placed_name for `TFun` then?

Reply:
Not sure... it's kind of expected to lose some information when going typed -> untyped.

Reply:
Maybe we should make `namePos` optional for now, because for old syntax it doesn't even make sense...

Reply:
But the entire type is new, so why does it matter?

Reply:
Because there will be a lot of `null_pos` positions encoded for the old syntax. Maybe an optional field or a `Null<Position>` would be better for performance.

Reply:
I doubt that would even be measurable... but if it makes you feel better, I don't mind.